xeon-phi: update drivers/infiniband's Kconfig and Makefile to

allow compilation of CCL-Direct (ibp)
---
diff -ruN a8/drivers/infiniband/hw/Makefile a9/drivers/infiniband/hw/Makefile
--- a8/drivers/infiniband/hw/Makefile	2015-09-10 09:35:14.874958188 -0700
+++ a9/drivers/infiniband/hw/Makefile	2015-09-10 09:35:42.139932498 -0700
@@ -10,3 +10,4 @@
 obj-$(CONFIG_INFINIBAND_NES)		+= nes/
 obj-$(CONFIG_INFINIBAND_OCRDMA)		+= ocrdma/
 obj-$(CONFIG_INFINIBAND_USNIC)		+= usnic/
+obj-$(CONFIG_INFINIBAND_SCIF)		+= scif/
diff -ruN a8/drivers/infiniband/Kconfig a9/drivers/infiniband/Kconfig
--- a8/drivers/infiniband/Kconfig	2015-09-10 09:35:14.875958141 -0700
+++ a9/drivers/infiniband/Kconfig	2015-09-10 09:35:42.139932498 -0700
@@ -55,6 +55,9 @@
 source "drivers/infiniband/hw/nes/Kconfig"
 source "drivers/infiniband/hw/ocrdma/Kconfig"
 source "drivers/infiniband/hw/usnic/Kconfig"
+source "drivers/infiniband/hw/scif/Kconfig"
+
+source "drivers/infiniband/ibp/Kconfig"
 
 source "drivers/infiniband/ulp/ipoib/Kconfig"
 
diff -ruN a8/drivers/infiniband/Makefile a9/drivers/infiniband/Makefile
--- a8/drivers/infiniband/Makefile	2015-09-10 09:35:14.881958235 -0700
+++ a9/drivers/infiniband/Makefile	2015-09-10 09:35:42.140932680 -0700
@@ -1,3 +1,4 @@
 obj-$(CONFIG_INFINIBAND)		+= core/
 obj-$(CONFIG_INFINIBAND)		+= hw/
 obj-$(CONFIG_INFINIBAND)		+= ulp/
+obj-$(CONFIG_IBP_SERVER)		+= ibp/
